lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -88,6 +89,7 @@
   private final ContainerRequestCounter containerRequestCounter;
   private final DriverStatusManager driverStatusManager;
   private final String trackingUrl;
+  private final LocalAddressProvider addressProvider;
```

Comment:
Yes, but keeping addressProvider reference would give more flexibility if we would like to access more stuff from it later.